parm/config/gfs/config.prepoceanobs

```diff
@@ -7,15 +7,20 @@ echo "BEGIN: config.prepoceanobs"
 export OCNOBS2IODAEXEC=${HOMEgfs}/sorc/gdas.cd/build/bin/gdas_obsprovider2ioda.x
 
 export SOCA_INPUT_FIX_DIR=@SOCA_INPUT_FIX_DIR@
-export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/soca/obs/config
+
+export OBS_YAML_DIR=${OBS_YAML_DIR:-${PARMgfs}/gdas/soca/obs/config}
 export OBSPREP_YAML=@OBSPREP_YAML@
 export OBS_LIST=@SOCA_OBS_LIST@
-[[ -n "${OBS_LIST}" ]] || export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/soca/obs/obs_list.yaml
+export OBS_LIST=${OBS_LIST:-${PARMgfs}/gdas/soca/obs/obs_list.yaml}
```

Comment:
This line no longer makes sense, as line 13 will define `OBS_LIST` and line 14 will set it to itself.

Reply:
@guillaumevernieres Is there a reason this is the way it is?

Reply:
removed